SwissDataScienceCenter/renku-ui
# Transitional landing

Fix #1374 

# Appearance

![image](https://user-images.githubusercontent.com/1196411/121901661-e1762600-cd26-11eb-983a-833b1a62f1d2.png)

![image](https://user-images.githubusercontent.com/1196411/121901756-f3f05f80-cd26-11eb-8ef1-b3aab5fe98ee.png)

![image](https://user-images.githubusercontent.com/1196411/121901923-2437fe00-cd27-11eb-92de-4a39dc6ca0ce.png)

# Testing

It is probably best to test on https://sekhar.dev.renku.ch/, where it is possible switch between the new and old UIs.

- Check the text and images are a good explanation and illustration of the new features
- Check that switching between the new and old UIs works
